base/applications/mshta/main.c

```diff
@@ -0,0 +1,19 @@
+/*
+ * PROJECT:         HTML Application Viewer
+ * LICENSE:         GPL - See COPYING in the top level directory
+ * FILE:            base/applications/mshta/main.c
+ * PURPOSE:         HTML Application Host
+ * PROGRAMMERS:     Jared Smudde (computerwhiz02@hotmail.com)
+ */
+
+#define WIN32_LEAN_AND_MEAN
+#include <stdio.h>
+#include <windows.h>
```

Comment:
Instead of these 3 above lines, just do:
```c
#include <windef.h>
#include <winbase.h>
#include <winuser.h>
```
Seeing how simple this mshta application is, nothing else should be required.